toolchain/check/import_cpp.cpp

```diff
@@ -44,6 +44,82 @@ static auto GenerateCppIncludesHeaderCode(
   return code;
 }
 
+namespace {
+
+// Used to convert Clang diagnostics to Carbon diagnostics.
+class CarbonClangDiagnosticConsumer : public clang::DiagnosticConsumer {
+ public:
+  // Creates an instance with the location that triggers calling Clang.
+  CarbonClangDiagnosticConsumer(Context& context, SemIRLoc loc)
```

Comment:
```suggestion
  // Creates an instance with the location that triggers calling Clang. `context` is required.
  explicit CarbonClangDiagnosticConsumer(Context* context, SemIRLoc loc)
```

Style, per https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md:

- explicit constructors: "Always mark constructors explicit unless there's a specific reason to support implicit or {} initialization."
- Pointer: "If it is captured and must outlive the call expression itself, use a pointer and document that it must not be null"

Reply:
Done.